### PR TITLE
[Node] Transactions metrics

### PR DESCRIPTION
**Pull request by mnmallea**
## Summary
- Added method to `ElasticSearchService` to get transaction stats from Elastic APM.
- Getting only production API metrics. (is needed dev and stage?)
- Added metrics to body posted to backend metrics API when are present.

## Known Issues
None

## Trello Card
???
[Change!] Link to the associated Trello card.

### CODE REVIEW

File: node_js/metrics/crashes_checks/services/ElasticSearchService.js [hunk 1]
```diff
@@ -29,5 +33,43 @@ module.exports = {
     };
     const url = `/*-error-*/_count`;
     return api.post(url, body);
-  }
+  },
+  getTransactionMetrics: (projectName, environment) =>
+    api.post('/apm-*-transaction-*/_search?size=0', {
```
Comment (juanpablo-rojas) [inline node_js/metrics/crashes_checks/services/ElasticSearchService.js:38]: This body is getting larger than expected. You could change the call to be similar to the environmentErrors: `api.post(url, body)` and segment the body into smaller pieces as you made with the `projectFilters` piece. Compose the body with all the small json pieces

File: node_js/metrics/crashes_checks/services/ElasticSearchService.js [hunk 2]
```diff
@@ -29,5 +33,43 @@ module.exports = {
...
+      aggs: {
+        requests: {
+          filter: {
+            bool: {
+              filter: [
+                ...projectFilters(projectName, environment),
+                {
+                  range: {
```
Comment (juanpablo-rojas) [inline node_js/metrics/crashes_checks/services/ElasticSearchService.js:46]: This range is repeated in both requests to the service, as said in other comment, you could extract just the range into a smaller piece and call it in every method just as `range: weekAgo()` or `range: monthAgo()`

File: node_js/index.js [hunk 3]
```diff
@@ -24,17 +28,37 @@ const ENV_BRANCH = 'development';
...
+  const repository = args.repository || args.r || '';
   return {
-    repository: args.repository || args.r || '',
+    repository,
     tech: args.tech || args.t || NODE_TECH,
     projectName: args.projectName || args.p || '',
     branch: args.branch || args.b || ENV_BRANCH,
-    metricsUrl: args.metricsUrl || args.m || NODE_METRICS_URL
+    metricsUrl: args.metricsUrl || args.m || NODE_METRICS_URL,
+    elasticApmProject: args['elastic-apm-project'] || repository
```
Comment (blacksam07) [inline node_js/index.js:38]: how you pass this args when you run the script?, I recommended passing a shorthand similar to`eap`

Comment (mnmallea) [reply node_js/index.js:38]: `--elastic-apm-project=<project name>`

Comment (mnmallea) [reply node_js/index.js]: Shorthand must contain only one letter :(

**Pull request merged**